lib/server/routes/health.js

```diff
@@ -23,7 +27,7 @@ HealthRouter.prototype.health = function(req, res) {
 
 HealthRouter.prototype._definitions = function() {
   return [
-    ['GET', '/health', this.health]
+    ['GET', '/health', this.getLimiter(limiter(1000)), this.health]
```

Comment:
Let's remove the limiter on this for now, and consider adding a token to use the endpoint in another PR.